nodejs/awsx/autoscaling/autoscaling.ts

```diff
@@ -16,63 +16,77 @@ import * as aws from "@pulumi/aws";
 import * as pulumi from "@pulumi/pulumi";
 
 import * as x from "..";
-import * as utils from "./../utils";
+import * as utils from "../utils";
 
 import { AutoScalingLaunchConfiguration, AutoScalingLaunchConfigurationArgs } from "./launchConfiguration";
 import { cronExpression, ScheduleArgs } from "./schedule";
 import * as stepScaling from "./stepScaling";
 import * as targetTracking from "./targetTracking";
 
 export class AutoScalingGroup extends pulumi.ComponentResource {
-    public readonly vpc: x.ec2.Vpc;
+    public readonly vpc!: x.ec2.Vpc;
```

Comment:
because we're no longer assigning to these fields in the constructor, we need `!` to satisfy TS.